cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Copyright (C) 2021 Bosch.IO GmbH
```

Comment:
The commit message needs to be adjust WRT the `--archive` or `--archive-all` options.